.github/workflows/checks.yaml

```diff
@@ -0,0 +1,12 @@
+name: checks
+
+on: [pull_request]
+
+jobs:
+  coding-style:
+    name: Code Formatting Check
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Run clang-format style check.
+        uses: jidicula/clang-format-action@v4.4.1
```

Comment:
We can specify a clang-version like
```yaml
steps:
  ...
  uses: jidicula/clang-format-action@v4.4.1
  with:
    clang-format-version: '13'
```
This makes sure the check won't break when the default version of clang-format changes.

Reply:
BTW, a `fallback-style` can also be specified.

Reply:
> We can specify a clang-version like
> 
> ```yaml
> steps:
>   ...
>   uses: jidicula/clang-format-action@v4.4.1
>   with:
>     clang-format-version: '13'
> ```
> 
> This makes sure the check won't break when the default version of clang-format changes.

Would clang-format introduce any breaking changes in the future? I am not sure. I thought using the latest version would be helpful so that we do not have to keep checking the latest clang-format version available and updating this file.



> BTW, a `fallback-style` can also be specified.

This means allowing an alternative style as a fallback which would break consistency and I'd prefer sticking to a single style.

Reply:
If we don't specify a clang-format version, developers won't know which version we use. They may use some old versions locally, thinking the code is fine, but get rejected by Github actions.

Reply:
I think the developers don't know the version that is checked by Gerrit at Chromium code reviews, too. Personally, I did not know it, I was just using my local tools and hoping for the best. I think they should use the latest version. In addition, I think version differences won't affect the formating itself. In other words, I expect all clang-format versions to be backwards compatible since we are not using them for complicated operations but just using a list of rules that was defined long time ago.